frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -1112,6 +1134,57 @@ pub mod pallet {
 			<QueuedSolution<T>>::put(solution);
 			Ok(())
 		}
+
+		/// Rotates the round which results in setting the current phase to
+		/// `Phase::Off` and removes all the current solutions.
+		///
+		/// The dispatch origin for this call must be `ForceOrigin`.
+		#[pallet::call_index(5)]
+		#[pallet::weight(({
+			if current_phase.is_signed() {
+				T::WeightInfo::force_rotate_round_from_signed()
+			}else {
+				T::WeightInfo::force_rotate_round()
+			}
+		}, DispatchClass::Operational))]
+		pub fn force_rotate_round(
+			origin: OriginFor<T>,
+			current_phase: Phase<T::BlockNumber>,
+		) -> DispatchResult {
+			T::ForceOrigin::ensure_origin(origin)?;
+			ensure!(current_phase == Self::current_phase(), <Error<T>>::CallNotAllowed);
+
+			Self::do_force_rotate_round();
+
+			Ok(())
+		}
+
+		/// Initializes an open signed phase. If the current phase is not
+		/// `Phase::Off` proceeds to the next round.
+		///
+		/// The dispatch origin for this call must be root.
+		#[pallet::call_index(6)]
+		#[pallet::weight((T::DbWeight::get().writes(1), DispatchClass::Operational))]
+		pub fn force_start_signed_phase(origin: OriginFor<T>) -> DispatchResult {
+			ensure_root(origin)?;
+
+			<ForcePhase<T>>::put(Phase::Signed);
+			Ok(())
+		}
+
+		/// Sets the current phase to `Phase::Emergency` and removes the current
+		/// queued solution.
+		///
+		/// The dispatch origin for this call must be `ForceOrigin`.
+		#[pallet::call_index(8)]
+		#[pallet::weight((T::DbWeight::get().writes(2), DispatchClass::Operational))]
+		pub fn force_start_emergency_phase(origin: OriginFor<T>) -> DispatchResult {
+			T::ForceOrigin::ensure_origin(origin)?;
+
+			<QueuedSolution<T>>::kill();
+			<CurrentPhase<T>>::put(Phase::Emergency);
```

Comment:
May be we can keep consistent flow for all force transition. That is:
- First update `ForcePhase`.
- Next block `on_initialise`, the actual transition takes place.

Could also be mentioned in the doc then that the changes will happen in the next block.

Reply:
Makes sense, I made it so we transition `on_initialize`.